### PR TITLE
MUZIMA-497 - Expanded cohort and delta sync

### PR DESCRIPTION
**Pull request by vshl**
Signed-off-by: Vishal Ravi Shankar <rsh.vishal@gmail.com>

API changes to accomodate the cohort membership resource and delta synchronization.

### CODE REVIEW

File: src/main/java/com/muzima/api/dao/CohortMembershipDao.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
```
Comment (mssavai) [inline src/main/java/com/muzima/api/dao/CohortMembershipDao.java]: Please update license information as per this ticket: https://tickets.muzima.org/browse/MUZIMA-518

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.muzima</groupId>
     <artifactId>muzima-api</artifactId>
-    <version>1.1.2</version>
+    <version>1.1.3</version>
```
Comment (mssavai) [inline pom.xml]: It's not known in which future version the merging into master will happen, so might need to revert this change to avoid future conflicts

**Comment by vshl** [comment]
@mssavai I created the branch as you mentioned. Should I create another pull request for it?

**Comment by vshl** [comment]
@mssavai Oh wait, I guess I misinterpreted your comment. Do you want me to create another branch with the new name and leave this alone?

**Comment by vshl** [comment]
@mssavai I understood what had to be done. I hope this is correct.

**Pull request merged**